app/templates/index.hbs

```diff
@@ -0,0 +1,5 @@
+<h1>REMINDERS</h1>
+
+{{#each model as | reminder |}}
+  <ul><li class='spec-reminder-item'>{{reminder}}</li></ul>
```

Comment:
The <ul> tag should be outside the {{each}}. Currently you are making 5 <ul> with a single <li> in each.